core/test-runtime/src/lib.rs

```diff
@@ -429,11 +466,19 @@ cfg_if! {
 				fn fail_on_wasm() -> u64 {
 					panic!("Failing because we are on wasm")
 				}
+
+				fn benchmark_indirect_call() -> u64 {
+					(0..10000).fold(0, |p, i| p + BENCHMARK_ADD_ONE.0.get()(i))
```

Comment:
we can definitely do this without the `SyncCell` thing, right? or maybe put it within this function

Reply:
I reworked this, to use the stuff from `sr-io`. This is only used in a benchmark, that I had written before doing these modifications.